client/src/components/byline-profiles.js/index.js

```diff
@@ -0,0 +1,263 @@
+/*
+ * Byline Profiles UI.
+ */
+
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import {
+  SortableContainer,
+  SortableElement,
+  arrayMove,
+} from 'react-sortable-hoc';
+import Autocomplete from 'react-autocomplete';
+import PropTypes from 'prop-types';
+import { Button } from '@wordpress/components';
+
+// Hooks.
+import useDebounce from '../../services/use-debounce';
+
+// Proptypes.
+import BYLINE_PROFILE_SHAPE from '../../../config/prop-types';
+
+const SortableItem = SortableElement(({
+  count,
+  bylineId,
+  name,
+  image,
+  removeItem,
+  removeAuthorLabel,
+}) => (
+  <li className="byline-list-item">
+    <input
+      type="hidden"
+      name={`byline_entry[${count}][type]`}
+      value={bylineId ? 'byline_id' : 'text'}
+    />
+    <input
+      type="hidden"
+      name={`byline_entry[${count}][value]`}
+      value={bylineId || name}
+    />
+    { image && <img src={image} alt={name} /> }
```

Comment:
```suggestion
    { image ? <img src={image} alt={name} /> : null }
```
seems to be the fashion these days.

Reply:
Yup, based on this article: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx